preconditions-assertj/src/main/resources/META-INF/services/org.assertj.core.presentation.Representation

```diff
@@ -1 +0,0 @@
-com.palantir.logsafe.testing.LoggableArgRepresentation
```

Comment:
Hang on, so what mechanism does assertj now use to discover this stuff if we've deleted this META-INF one?

Reply:
This PR changes the implementation to explicitly use the representation for the logsafe assertions.

Reply:
The only other non-deprecated option is to explicitly register it in your test classes, which doesn't work here because this is a library.